### DISCUSSION TITLE
Basic auth is not working

### DISCUSSION THREAD

**Discussion by surajit-zs** [gofr-dev/gofr#461] [Q&A] [answered]
https://github.com/gofr-dev/gofr/blob/b1e7edcb10ae8689cbb8e1fcdbf4eb6e5cf26da0/pkg/gofr/http/middleware/basic_auth.go#L28

the issue is with **basic** i think it will be **Basic**
```go
			authParts := strings.Split(authHeader, " ")
			if len(authParts) != 2 || authParts[0] != "Basic" {
				http.Error(w, "Unauthorized: Invalid Authorization header", http.StatusUnauthorized)
				return
			}
```

**Comment by surajit-zs** [comment]
<img width="729" alt="image" src="https://github.com/gofr-dev/gofr/assets/97510117/f08a884a-2fd5-47e7-8875-7522fd940df7">

is this expected.

**Reply by srijan-27** [reply]
Issue is being tracked here: https://github.com/gofr-dev/gofr/issues/471

**Comment by srijan-27** [comment, marked as answer]
@surajit-zs - Thanks for reporting the issues. These are fixed in #464 and #476 respectively.
